CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ Changelog
 
 ### Bugfixes
 
+- Updated `examples/meyer_penny_game.py` with the correct path to the Meyer
+  Penny game exercise in `docs/source/exercises.rst` (gh-1045, @appleby).
```

Comment:
Is there a preferred line length for this file? I'm attempting to match the entry in `Improvements and Changes` just above, which looks like 80 cols, but the `Improvements and Changes` section for the last release looks to be slightly wider.

Reply:
100 cols is our line length limit -- i guess i just made the above one shorter